### DISCUSSION TITLE
About me | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#118] [Announcements]
https://blog.goalonez.site/aboutme.html 

 Goalonez Blog

**Comment by yizhitangtongxue** [comment]
博客真好看，用的什么搭建的😀

**Comment by Goalonez** [comment]
> @yizhitangtongxue
> 博客真好看，用的什么搭建的😀

可以参考这篇blog

https://blog.goalonez.site/blog/VitePress%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA.html

**Comment by n4android** [comment]
大佬  求一个 follow 邀请码

**Comment by oddity123** [comment]
您好:
	我在网上看到了您的博客, 写的非常棒.
	我估计您也是一个程序员, 我想请教怎么快速关闭Fn键. 平时不写代码的时候, 使用F1等特殊功能键, 如降低音量等操作, 但是在我使用IDEA的时候, 我希望使用的是F9, 比如debug的时候会使用到. mac设置里确实能调整, 但是每次都需要去设置里调整, 路径为: 系统设置-键盘-键盘快捷键-fn功能键. Win上可以fn+esc键快速开启和关闭, 请问您有办法实现这个操作吗, 我使用的是mac上自带的自动流程, 但是失败了.
	如果能解决此问题, 希望能得到您的回复, 谢谢.🙏

**Comment by Goalonez** [comment]
> @oddity123
> 您好:
> 	我在网上看到了您的博客, 写的非常棒.
> 	我估计您也是一个程序员, 我想请教怎么快速关闭Fn键. 平时不写代码的时候, 使用F1等特殊功能键, 如降低音量等操作, 但是在我使用IDEA的时候, 我希望使用的是F9, 比如debug的时候会使用到. mac设置里确实能调整, 但是每次都需要去设置里调整, 路径为: 系统设置-键盘-键盘快捷键-fn功能键. Win上可以fn+esc键快速开启和关闭, 请问您有办法实现这个操作吗, 我使用的是mac上自带的自动流程, 但是失败了.
> 	如果能解决此问题, 希望能得到您的回复, 谢谢.🙏

是指平时用还是当特殊功能键，打开IDEA的时候自动切成F1、F2吗？
自动切换我没研究过，不太清楚有什么方案可以做到。
我是直接就设置成默认F1、F2了，特殊功能键才通过Fn去触发。因为我日常使用几乎不用特殊功能键。大部分的键盘操作都通过Raycast和软件自带的快捷键设置来处理。

**Comment by Fakexiaoer** [comment]
大佬，您好，看您博客挺久了，是否方便加个联系方式？

**Reply by Fakexiaoer** [reply]
有点冒昧，但真的非常希望跟您学习。

**Reply by Goalonez** [reply]
首页有联系方式的，可以通过那几个方式联系我
